it/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/services/SoftwareChoicesServiceISpec.scala

```diff
@@ -40,6 +41,20 @@ class SoftwareChoicesServiceISpec extends PlaySpec with GuiceOneServerPerSuite {
     }
   }
 
+  Seq("software-vendors.json") foreach { file =>
+    file must {
+      "have url starting with https://" in {
+        Try{
+          val test = app(file).injector.instanceOf[SoftwareChoicesService].softwareVendors.vendors
+          test.foreach{ element =>
+            element.url.startsWith("https://") shouldBe true
```

Comment:
```suggestion
            element.url.startsWith("https://") shouldBe true
            element.website.startsWith("https://") shouldBe true
```
we have two urls in the vendor model which we should check, lets test them both for now, i've raised a ticket to remove one of them

Reply:
updated website test